play/src/containers/GameInProgressContainer.tsx

```diff
@@ -71,6 +71,14 @@ export function GameInProgressContainer(props: GameInProgressContainerProps) {
     }
```

Comment:
Play is updated to show a leaderboard screen when prequestion configuration options are being selected on the teacher side